projects_index.yaml

```
@@ -352,6 +352,156 @@
     en: Official implementation of "ZeroQ:A Novel Zero Shot Quantization Framework"
     zh: ZeroQ:A Novel Zero Shot Quantization Framework官方实现
 
+- repo_url: https://github.com/Gavince/MTL
+  paper_url: ''
+  type: Community Implementation
+  mmrepos:
+    - MMClassification
+  tags:
+    - Multi-Task Learning
+    - CTR
+  summary:
+    en: Implementation of Multi-task Learning Algorithms in Recommendation System
+    zh: 推荐系统之多目标学习
+
+- repo_url: https://github.com/likyoo/open-cd
+  paper_url: ''
+  type: Library
+  mmrepos:
+    - MMClassification
+    - MMSegmentation
+  tags:
+    - Change Detection
+    - Toolbox
+  summary:
+    en: An open source change detection toolbox based on a series of open source general vision task tools
+    zh: 变化检测开源工具箱
+
+- repo_url: https://github.com/chufengt/Visual-Recognition-by-Request
+  paper_url: https://arxiv.org/abs/2207.14227
+  type: Official Implementation
+  mmrepos:
+    - MMClassification
+  tags:
+    - Visual Recognition
+  summary:
+    en: Code for the paper "Visual Recognition by Request"
+    zh: Visual Recognition by Request 官方实现
+
+
+- repo_url: https://github.com/linjing7/VR-Baseline
+  paper_url: ''
+  type: Library
+  mmrepos:
+    - MMEditing
+    - MMClassification
+  tags:
+    - Video Restoration
+  summary:
+    en: Video Restoration Toolbox including FGST (ICML 2022) and S2SVR (ICML 2022)
+    zh: Video Restoration 工具箱
+
+- repo_url: https://github.com/ngfuong/phishing-detector
+  paper_url: ''
+  type: Demo
+  mmrepos:
+    - MMClassification
+  tags:
+    - Video Restoration
+  summary:
+    en: A Heroku-powered web application that supports phishing detection of malicious websites.
+    zh: 一个支持phishing detection的web应用
```

```suggestion
    zh: 一个支持 phishing detection 的 web 应用
```

Better to add sapce between Chinese character and English letter, the same comments about other parts